spec/support/user_authentication.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+def login_as_user(user = Fabricate(:user))
```

Comment:
To reduce duplication on `user` word 🙏 
```suggestion
def login_as(user = Fabricate(:user))
```